.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+##### Description of the problem 
+
+
```

Comment:
I think I would add `...` here. Markdown seem to be ignoring the line breaks otherwise.